src/models/room.model.ts

```diff
@@ -38,6 +38,22 @@ class Room
             }
         });
     }
+
+    static updateById = (data: RoomData, callback: Callback) => {
+        sql.query(
+            'UPDATE rooms SET ? WHERE id = ?',
+            [{ title: data.title, body: data.body }, data.id],
```

Comment:
```suggestion
            [data, data.id],
```
This works, since id is same, we needn't worry.